### ISSUE TITLE
First introduction of comments

### ISSUE THREAD

**Issue by zerpiko** [swcarpentry/python-novice-inflammation#867]
Hi. 

I noticed that in lesson 2 (**Analyzing Patient Data**) the use of **#** to comment out text is explained with:

>Everything in a line of code following the ‘#’ symbol is a comment that is ignored by Python. Comments allow programmers to leave explanatory notes for other programmers or their future selves.

after using a comment in an example. However, the first use of comments is in lesson 1 (**Python Fundamentals**)
```
# There are 2.2 pounds per kilogram
weight_lb = 2.2 * weight_kg
print(weight_kg_text, weight_kg, 'and in pounds:', weight_lb)
```

Wouldn't be better to move the explanation (and associated key point) to the first lesson?

**Comment by maxim-belkin** [comment]
Good catch, @zerpiko! I think we should do that.
Do you agree, @ldko and @annefou?

**Comment by ldko** [comment]
Yes, I think @zerpiko makes a good point. If Jose wants to open a PR to move the explanation, I think that would be great!

**Comment by annefou** [comment]
Yes I agree,

**Comment by zerpiko** [comment]
Hi, thanks for the feedback. I created a PR for this issue.
